src/test/scala/uk.gov.nationalarchives.fileformat/FileUtilsTest.scala

```diff
@@ -0,0 +1,181 @@
+package uk.gov.nationalarchives.fileformat
+
+import java.nio.file.{Path, Paths}
```

Comment:
`..., Paths}` appears to be an unnecessary import

Reply:
Yep, I've taken it out.